test/functional/page_objects/settings_page.ts

```diff
@@ -453,19 +453,19 @@ export class SettingsPageObject extends FtrService {
     await this.common.scrollKibanaBodyTop();
 
     // if flyout is open
-    const flyoutView = await this.testSubjects.exists('createIndexPatternButtonFlyout');
+    const flyoutView = await this.testSubjects.exists('createDataViewButtonFlyout');
```

Comment:
curious if we should consider creating a constant for this in `@kbn/shared-ux-components` or something, in a follow-up.